.github/workflows/validate_datasets.yml

```diff
@@ -111,4 +111,4 @@ jobs:
       env:
         # When proposing new features, schema changes may be necessary.
         # Update this URL to the schema.json from PRs to the spec, when needed.
-        BIDS_SCHEMA: https://bids-specification.readthedocs.io/en/latest/schema.json
+        BIDS_SCHEMA: https://bids-specification--1884.org.readthedocs.build/en/1884/schema.json
```

Comment:
unrelated to this PR but this little 'trick' to test a version of the schema related to a PR in the spec should be mentioned in our doc, so that BEP leads know how to start testing their datasets

Reply:
[#1864](https://github.com/bids-standard/bids-specification/pull/1864) was merged so this PR should be updated